fli-iam/shanoir-ng
# [fli-iam#1859] Add an API to fetch subject names by ids

Introduce a new API (POST /studies/subject/names) to fetch IdNames from a list of subject ids. 

How to test :
- Display dataset details in view, edit mode
- Display study in edit, create mode
- Display a list of datasets in VIP execution view

=> Excluding dataset edit and study edit/create, subject names should be fetch only from the new API
